### ISSUE TITLE
crash in BList::CountItems() on Alt+q

### ISSUE THREAD

**Issue by diversys** [humdingerb/quicklaunch#12]
```
Debug information for team /boot/system/apps/QuickLaunch/QuickLaunch (18007):
CPU(s): 2x Intel Core™ i7-3635QM
Memory: 1022.94 МБ total, 470.44 МБ used
Haiku revision: hrev51120 Apr 23 2017 18:55:31 (x86_64)

Active Threads:
	thread 18007: QuickLaunch (main)
	thread 18030: w>QuickLaunch: Открыть 
	thread 18036: w>Настройки 
	thread 18726: team 18007 debug task 
	thread 18035: w>QuickLaunch 
		state: Exception (Segment violation)

		Frame		IP			Function Name
		-----------------------------------------------
		0x7f6efb7e88c0	0xcc52d62f1	BList::CountItems() const + 0x1 
			Disassembly:
				BList::CountItems() const:
				0x0000000cc52d62f0:               55  push %rbp
				0x0000000cc52d62f1:           8b4714  mov 0x14(%rdi), %eax <--

			Frame memory:
				[0x7f6efb7e88b0]  ..~.n...........   10 8c 7e fb 6e 7f 00 00 af e3 e1 ed 0a 01 00 00
		0x7f6efb7e8c20	0x10aede1e3aa	MainWindow::BuildList() + 0x4ea 
		0x7f6efb7e8da0	0x10aede1ee50	MainWindow::MessageReceived(BMessage*) + 0x6f0 
		0x7f6efb7e8fd0	0xcc5274279	BWindow::DispatchMessage(BMessage*, BHandler*) + 0x379 
		0x7f6efb7e9060	0xcc5270023	BWindow::task_looper() + 0x183 
		0x7f6efb7e9080	0xcc51ae306	BLooper::_task0_(void*) + 0x26 
		0x7f6efb7e90a0	0x1d61667c3b0	thread_entry + 0x10 
		00000000	0x7fcbd7f5f260	commpage_thread_exit + 0 
```

**Comment by humdingerb** [comment]
As determined on IRC, this may be a bug that only shows on 64bit Haiku. Need to investigate...

**Comment by humdingerb** [comment]
Still cannot reproduce. But for diver the bug only happens when the option "Remember last search item" is on.

**Comment by humdingerb** [comment]
@diversys : Can you attach a full crash report?

**Comment by diversys** [comment]
[QuickLaunch-7738-debug-29-04-2017-08-46-26.txt](https://github.com/humdingerb/quicklaunch/files/966004/QuickLaunch-7738-debug-29-04-2017-08-46-26.txt)

**Comment by humdingerb** [comment]
@diversys, can you test this version, please:
[QuickLaunch_test.zip](https://github.com/humdingerb/quicklaunch/files/974116/QuickLaunch_test.zip)

**Comment by diversys** [comment]
Is it a 32bit binary? I'm on x86_64.

**Comment by humdingerb** [comment]
oh, sorry... Try this: 
[QuickLaunch_test_64bit.zip](https://github.com/humdingerb/quicklaunch/files/974142/QuickLaunch_test_64bit.zip)

**Comment by diversys** [comment]
Yep, this version doesn't crash. Thanks! :)

**Comment by humdingerb** [comment]
Thanks for testing! I'm glad that worked...

**Comment by humdingerb** [comment]
Fixed with 7f8f535a080a543439cf9f1a301f26ff68539bd8
